### ISSUE TITLE
yolo4 custom object detection compression

### ISSUE THREAD

**Issue by samohadid** [IntelLabs/distiller#555]
I want to thank you for your amazing work. 
from a week ago I read your work, checked all issues, and the pull requests and I was still confused. as mentioned In the title Iam trying to deep compress yolo4 weights file. I think that compressing a weight file is the easiest approach for the problem.
 After I read the paper "deep compression: compressing deep neural networks with pruning, trained quantization and Huffman coding"  . I started searching for a reliable implementation. I Found that your work is the most famous and have covered many application.
 The authors of the paper said that after pruning the weights file they train the network and re prune if needed.
I want to deep compress yolo4 custom object detection. 
I found that you included all the pyimage models natively implemented in Pytorch. I found an implantation for yolo4 in Pytorch yet I really think that avoiding to natively add darknet is the wisest approach  
I only need to compress the weight file. since darknet is vast and complicated Implementing or merging the two repositories is not an efficient approach and Iam really concerned about Pytorch not supporting open cv about the accuracy.
I need to confirm that this an implementation of the paper mentioned above.
And a way to compress a custom yolo4 network favorable a stand alone way to compress any weight file (doesn't need to be yolo we can transform form yolo to other formats and vise versa) and make the training in the original framework.
